packages/react-core/src/demos/CardDemos.md

```diff
@@ -413,3 +416,572 @@ CardDetailsDemo = () => {
+              <FlexItem>
+                <a href="#">Image Vulnerabilities</a>
+              </FlexItem>
+              <FlexItem>
+                <span style={{ color: '#8a8d90' }}>0 vulnerabilities</span>
+              </FlexItem>
+            </Flex>
+          </Flex>
+        </GridItem>
+      </Grid>
+    </CardBody>
+  );
+
+  const drawerTitle = (
+    <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+      <FlexItem spacer={{ default: 'spacerMd' }}>
+        <span>Notifications</span>
+      </FlexItem>
+      <Label color="red" icon={<ExclamationCircleIcon />}>
+        1
+      </Label>
+      <Label color="orange" icon={<ExclamationTriangleIcon />}>
+        3
+      </Label>
+      <Label color="green" icon={<CheckCircleIcon />}>
+        3
+      </Label>
+      <Label color="blue" icon={<ExclamationCircleIcon />}>
+        3
+      </Label>
+      <Label color="green" icon={<BellIcon />}>
+        3
+      </Label>
+    </Flex>
+  );
+
+  const drawer = (
+    <NotificationDrawer>
+      <NotificationDrawerBody>
+        <NotificationDrawerGroup
+          count={0}
+          onExpand={handleDrawerToggleClick}
+          isExpanded={drawerExpanded}
+          title={drawerTitle}
+        >
+          <NotificationDrawerList isHidden={!drawerExpanded}>
+            <NotificationDrawerListItem variant="danger">
+              <NotificationDrawerListItemHeader variant="danger" title="Critical alert regarding control plane" />
+              <NotificationDrawerListItemBody>
+                This is a long description to show how the title will wrap if it is long and wraps to multiple lines.
+              </NotificationDrawerListItemBody>
+            </NotificationDrawerListItem>
+            <NotificationDrawerListItem variant="warning">
+              <NotificationDrawerListItemHeader variant="warning" title="Warning alert" />
+              <NotificationDrawerListItemBody>
+                This is a warning notification description.
+              </NotificationDrawerListItemBody>
+            </NotificationDrawerListItem>
+          </NotificationDrawerList>
+        </NotificationDrawerGroup>
+      </NotificationDrawerBody>
+    </NotificationDrawer>
+  );
+
+  return (
+    <Card>
+      {header}
+      {body}
+      <Divider />
+      {drawer}
+    </Card>
+  );
+};
+```
+
+### Status Tabbed
+
+```ts
+import * as React from 'react';
+import { Card, CardBody, CardHeader, DescriptionList, DescriptionListDescription, DescriptionListGroup, DescriptionListTerm, Flex, FlexItem, Grid, GridItem, Spinner, Tab, TabContent, Tabs, TabTitleText, Title } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+const descriptionListData = [
+    {
+        status: 'Running',
+        resourceName: 'Resource name that is long and can wrap',
+        detail: '121 Systems',
+        icon: <Spinner size='md' aria-valuetext='Loading...'></Spinner>
```

Comment:
Let's replace this with a check icon

Reply:
They core demo has spinners.  Will that be updated as well?

Reply:
Changed the spinners to checks, leaving this conversation open for @tlabaj's  comment.